config/skills/preflight/SKILL.md

```
@@ -0,0 +1,79 @@
+---
+name: preflight
+description: Ground external prompts, issues, and specs against repository reality before implementation. Use when the user types /preflight, asks to preflight or sanity-check a task, or pastes a feature prompt/issue to interrogate triggers, state, and edge cases before touching code.
+---
+
+# Task Preflight & Spec Grounding
+
+## Use When
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)